e2e/test/scenarios/sharing/subscriptions.cy.spec.js

```diff
@@ -66,6 +66,19 @@ describe("scenarios > dashboard > subscriptions", () => {
     cy.findByText(/Share this dashboard with people *./i);
   });
 
+  describe("sidebar toggling behavior", () => {
+    it("should allow toggling the sidebar", () => {
+      openDashboardSubscriptions();
+
+      cy.findByLabelText("subscriptions").click();
+      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
+      cy.findByText("Email it").should("not.exist");
+      cy.findByLabelText("subscriptions").click();
+      // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
+      cy.findByText("Email it");
```

Comment:
Let's not use deprecated practices in new code. We expect `findByText` to be wrapped in `within` (more in #30059). But in this case, maybe we can assert that an `aside` disappears after we click the "subscriptions" button again? For example:

```js
sidebar().should("not.exist");
cy.findByLabelText("subscriptions").click();
sidebar().findByText("Email it").should("exist");
```

Reply:
Ah ok I see the better way to do it down below in the file, was looking at the tests up above. Easy to fix.